src/troute-network/troute/HYFeaturesNetwork.py

```diff
@@ -29,14 +28,19 @@ def find_layer_name(layers, pattern):
         if re.search(pattern, layer, re.IGNORECASE):
             return layer
     return None
-
+  
```

Comment:
minor whitespace diff here...

Reply:
oops, I must not have had black installed